pallets/msa/src/tests.rs

```diff
@@ -1968,40 +1999,6 @@ pub fn delegation_expired_long_back() {
 	})
 }
 
-#[test]
-pub fn ensure_all_schema_ids_are_valid_errors() {
```

Comment:
Were these tests supposed to be dropped?

Reply:
Rebase issue, brought PR back to normal